requirements.txt

```diff
@@ -1,2 +1,3 @@
 ovos-plugin-manager>=0.0.5
-piper-tts==1.2.0
+piper-tts==1.1.0
+onnxruntime==1.16.3
```

Comment:
onnxruntime is already a dependency of `piper-tts`; https://github.com/rhasspy/piper/blob/master/src/python_run/requirements.txt#L2

Reply:
I thought that should be the case, but installs within the Neon docker container were failing as that dependency wasn't installed